source/config/Config.cpp

```diff
@@ -539,6 +615,32 @@ string PlainConfig::LogConfig::ParseDeviceClientLogType(string value)
     }
 }
 
+string PlainConfig::LogConfig::StringifyDeviceClientLogLevel(int level) const
+{
+
+    switch (level)
+    {
+        case 0:
+            return "error";
```

Comment:
Should probably be ALL CAPS to be consistent with `PlainConfig::LogConfig::ParseDeviceClientLogLevel`.

Reply:
Yes, I agree, I was checking out the other configs defined in Config.cpp and they used this lower case version, but it SHOULD be upper.